source/_integrations/freebox.markdown

```diff
@@ -44,28 +57,23 @@ port:
   type: string
 {% endconfiguration %}
 
-You can find out your Freebox host and port by opening the address <http://mafreebox.freebox.fr/api_version> in your browser. The
-returned json should contain an api_domain (`host`) and a https_port (`port`).
-Please consult the [api documentation](https://dev.freebox.fr/sdk/os/) for more information.
-
 <div class='note warning'>
   
-If you change your Freebox router for a new one, you need to delete the `freebox.conf` file located in your Home Assistant configuration directory to make the association again.
+  If you change your Freebox router for a new one, go into your Home Assistant configuration `.storage` folder and delete the "freebox" folder, then add the component again.
```

Comment:
```suggestion
  If you change your Freebox router for a new one, go into your Home Assistant configuration `.storage` folder and delete the "freebox" folder, then add the integration again.
```

Besides, this sounds weird. Shouldn't the user just delete the integration via the UI and re-add it? The cleanup of the storage folder can be done automatically as part of the unload procedure right?

Reply:
That would be nice actually. I didn't know something was planned to clean up the storage folder.

Will see (and for the iCloud integration), not in the PR for now, will see and update here as soon as it's get coded 😉.

Maybe in a new PR perhaps, there is already a lot in this one.

Reply:
? I'm not saying it happens automatically, but there is an integration unload method that allows you do those things right?

I think we should never instruct the user to go into their .storage folder.

Reply:
I understood not automatically.

I thought the `helpers.storage.Store` has a function that can clear itself (depending on the `STORAGE_KEY` of course) and I could call this function on the `async_unload_entry` of the integration.

I completely agree that the user shouldn't go to the `.storage` folder (there is a reason why it is invisible).

In fact I've updated the location here but during my development and test, I didn't need to remove the conf file at all. Before, the token file was `freebox.conf`, now it's `f"{freebox_dir.path}/{slugify(self._host)}.conf"` : since the host will change with a new Freebox, I don't think this note is still necessary.